src/extension/background-script/db.ts

```diff
@@ -26,6 +26,9 @@ class DB extends Dexie {
     });
     this.version(3).stores({
       permissions: "++id,allowanceId,host,method,enabled,blocked,createdAt",
+      contacts: "++id, &lnAddress",
```

Comment:
I think you need to create a "new version" every time we add new "tables" to the DB. i.e.:
```ts
    this.version(4).stores({
      contacts: "++id, &lnAddress",
    })
```

Reply:
Should we maybe also add the current account that is being used here? I'm not sure whether a contact list should be something global or also on an account level. 🤔

Reply:
Yeah, that's a good question... I suppose for myself, I use the different accounts as just different sources of funding so I would want a global account list. However I can see the case if you want to hold accounts separately.

Reply:
Yeah, that's a good question... I suppose for myself, I use the different accounts as just different sources of funding so I would want a global account list. However I can see the case if you want to hold accounts separately

Reply:
Would it be as simple as adding an `accountId`? I've been a bit confused on the distinction between `db.ts` and `state.ts`

Reply:
> I think you need to create a "new version" every time we add new "tables" to the DB. i.e.:
> 
> ```ts
>     this.version(4).stores({
>       contacts: "++id, &lnAddress",
>     })
> ```

Looks like you need to add a new version even if you modify an existing table - would that be a problem if we want to add other fields to contacts table in the future?

Reply:
> Would it be as simple as adding an accountId? I've been a bit confused on the distinction between db.ts and state.ts

I think so, yes. You would add the `accountId` to the contacts table and reference the account that is currently being used as you store new lightning addresses. 

> Looks like you need to add a new version even if you modify an existing table - would that be a problem if we want to add other fields to contacts table in the future?

I don't think so. It's the same situation for all other tables that already exist. But if we think the accountId could be useful it might be better to include it right away.

Reply:
Sounds good, I added `accountId` to the schema